setup/ansible/roles/common_ubuntu/tasks/sys_deps.yml

```diff
@@ -48,14 +48,15 @@
   sudo: Yes
   apt: update_cache=yes
 
-- name: Install python Enthought dependencies
+- name: Install python dependencies
   sudo: True
   command: >
```

Comment:
Should we be using the 'apt' ansible module for this, rather than the generic 'command' module?